api/migrations/versions/7f3d601390cb_.py

```diff
@@ -0,0 +1,105 @@
+    sa.Column('ethnicity_name', sa.String(length=100), nullable=True),
+    sa.PrimaryKeyConstraint('user_ethnicity_id')
+    )
+    user_gender = op.create_table('user_gender',
+    sa.Column('user_gender_id', sa.Integer(), nullable=False),
+    sa.Column('gender_name', sa.String(length=10), nullable=True),
+    sa.PrimaryKeyConstraint('user_gender_id')
+    )
+    user_title = op.create_table('user_title',
+    sa.Column('user_title_id', sa.Integer(), nullable=False),
+    sa.Column('title_name', sa.String(length=10), nullable=True),
+    sa.PrimaryKeyConstraint('user_title_id')
+    )
+    op.add_column(u'app_user', sa.Column('affiliation', sa.String(length=255), nullable=True))
+    op.add_column(u'app_user', sa.Column('deleted_datetime_utc', sa.DateTime(), nullable=True))
+    op.add_column(u'app_user', sa.Column('department', sa.String(length=255), nullable=True))
+    op.add_column(u'app_user', sa.Column('firstname', sa.String(length=100), nullable=True))
+    op.add_column(u'app_user', sa.Column('is_deleted', sa.Boolean(), nullable=True))
+    op.add_column(u'app_user', sa.Column('lastname', sa.String(length=100), nullable=True))
+    op.add_column(u'app_user', sa.Column('nationality_id', sa.Integer(), nullable=True))
+    op.add_column(u'app_user', sa.Column('residence_id', sa.Integer(), nullable=True))
+    op.add_column(u'app_user', sa.Column('user_disability_id', sa.Integer(), nullable=True))
+    op.add_column(u'app_user', sa.Column('user_ethnicity_id', sa.Integer(), nullable=True))
+    op.add_column(u'app_user', sa.Column('user_gender_id', sa.Integer(), nullable=True))
+    op.add_column(u'app_user', sa.Column('user_title_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'app_user', 'user_gender', ['user_gender_id'], ['user_gender_id'])
+    op.create_foreign_key(None, 'app_user', 'country', ['nationality_id'], ['country_id'])
+    op.create_foreign_key(None, 'app_user', 'user_ethnicity', ['user_ethnicity_id'], ['user_ethnicity_id'])
+    op.create_foreign_key(None, 'app_user', 'country', ['residence_id'], ['country_id'])
+    op.create_foreign_key(None, 'app_user', 'user_disability', ['user_disability_id'], ['user_disability_id'])
+    op.create_foreign_key(None, 'app_user', 'user_title', ['user_title_id'], ['user_title_id'])
+    op.drop_index('unique_user_code', table_name='password_reset')
+    # ### end Alembic commands ###
+    op.bulk_insert(country, [{'country_id': 1, 'country_name': 'South Africa'}])
+    op.bulk_insert(user_category, [{'user_category_id': 1, 'name': 'Postdoc', 'description': 'Postdoc', 'group': 'Postdoc'}])
+    op.bulk_insert(user_disability, [{'user_disability_id': 1, 'disability_name': 'None'}])
+    op.bulk_insert(user_ethnicity, [{'user_ethnicity_id': 1, 'ethnicity_name': 'None'}])
+    op.bulk_insert(user_gender, [{'user_gender': 1, 'gender_name': 'Male'}])
+    op.bulk_insert(user_title, [{'user_title_id': 1, 'title_name': 'Mr'}])
```

Comment:
Not sure if this is the way to add static data to the tables? Also this is just dummy data for now. Once the correct way of doing this is confirmed, I'll fill in the rest of the static data using the existing Google Form template

Reply:
I'm sure it's fine, but I'd avoid setting the ids manually and rather make sure postgres is generating them. Also, I doubt you'd need to use a bulk insert, you could probably use the orm?

Reply:
@amritpurshotam Agreed that the bulk entry is the correct way for lots of data :)